src/backend/src/services/change-requests.services.ts

```diff
@@ -778,8 +778,7 @@ export default class ChangeRequestsService {
     });
 
     // send slack message to CR reviewers
-    newReviewers.forEach(async (user) => {
-      await sendSlackRequestedReviewNotification(user.userSettings!.slackId, changeRequestTransformer(foundCR));
-    });
+    const relevantThreads = await prisma.message_Info.findMany({ where: { changeRequestId: foundCR.crId } });
```

Comment:
because we're retrieving the message threads from the change request and sendSlackRequestedReviewNotification is already async, we may as well move this call into that method as well